crates/bevy_tasks/src/task_pool.rs

```diff
@@ -106,7 +111,8 @@ impl TaskPool {
 
         let executor = Arc::new(async_executor::Executor::new());
 
-        let num_threads = num_threads.unwrap_or_else(num_cpus::get);
+        //let num_threads = num_threads.unwrap_or_else(num_cpus::get);
```

Comment:
TODO: `num_cpus` does not support wasm (just returns 1). Submit a PR implementing https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.Navigator.html#method.hardware_concurrency

Reply:
`let num_threads = web_sys::window().unwrap().navigator().hardware_concurrency()` should work.

Reply:
Thanks for the snippet. Although what I wanted to say is that it would be better to implement this directly into `num_cpus` crate, which aims to be multi-platform.